content/posts/newsletter/2019-03-31.md

```diff
@@ -0,0 +1,46 @@
+---
+title: The XMPP Newsletter, 31st March 2019
+save_as: 2019/03/newsletter-31-march/index.html
+url: 2019/03/newsletter-31-march/
+date: 2019-02-28 00:00
+author: Seve
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter.
+
+If you have an article, tutorial or blog post you'd like us to include in the
+newsletter, please [submit it on the XMPP wiki](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+## News
+
+A design [job has been posted at the Open Source Design community](https://opensourcedesign.net/jobs/jobs/2019-03-19-design-of-badges-for-different-xmpp-compliance-levels) for Compliance Suite badges.
+It is an initiative that aims to create graphic elements for developers to use on their clients and servers to be able to identify compliance levels of such software.
+
+A [wiki page to keep track of integrations has been created at the XSF's Wiki](https://wiki.xmpp.org/web/Integrations). The idea behind it is to take a look at projects or services that support features like notifications but do not include XMPP as part of the integrations they support.
+This page can be used to follow missing integrations on all kind of software and make it visible so other people can work or promote them.
+
+<form style="padding: 10px; text-align:center; margin-bottom: 30px;"
+      action="https://tinyletter.com/xmpp" method="post" target="popupwindow"
+      onsubmit="window.open('https://tinyletter.com/xmpp', 'popupwindow',
+      'scrollbars=yes,width=800,height=600');return true">
+<p><label for="tlemail">Subscribe to receive the next edition in your inbox</label></p>
+<p><input type="text" placeholder="Email address" name="email" id="tlemail" /></p>
+<input type="hidden" value="1" name="embed"/>
+<input type="submit" style="padding: 10px; border-radius: 5%" value="Subscribe" />
+</form>
+
+
+## Software releases
+
+## Servers
```

Comment:
You probably want to put the subsections at ``###`` level instead of at ``##`` level.